src/app/home/home.component.ts

```diff
@@ -3,6 +3,7 @@ import {SwUpdate} from '@angular/service-worker';
 import {SubSink} from 'subsink';
 import {environment} from '../../environments/environment';
 import {BrowserParamsService} from '../browser-params.service';
+import { Router } from '@angular/router';
```

Comment:
Nit: format the file with: `npx prettier --write src/app/home/home.component.ts` (for each file)

Or use the Prettier VS Code extension :)

Reply:
Done. I had another one set up. Though it did fix a bunch of other lines that are now in the current pull request.